.git-blame-ignore-revs

```diff
@@ -246,6 +250,13 @@ a9778bafea2ee3e92c6671144fa08d1394f11099
 # Merge pull request #10352 from NREL/update-license-year-2024 [Edwin Lee, 2024-01-02]
 6fafbb763fe676012f5db10da7b3e8449f578094
 
+# Update the license year to 2025 [Jason W. DeGraw, 2025-01-01]
+5b2874d37a39aa528c7fba0d87b675d2d30cc146
+#   1053 files changed, 1059 insertions(+), 1056 deletions(-)
+
+# Merge pull request #10886 from NREL/update-license-year-2025 [Edwin Lee, 2025-01-08]
+b46ced64a420a7df09af0f301499d0548ac89966
```

Comment:
Update .git-blame-ignore-revs a bit, especially removing the commits that update the license file as they show on **every** blame you do otherwise